### PR TITLE
[Ldap] minor: better query string docs

### PR DESCRIPTION
**Pull request by Simperfit**
see https://github.com/symfony/symfony/issues/26316#issuecomment-374304364

### CODE REVIEW

File: security/ldap.rst [hunk 1]
```diff
@@ -313,12 +313,9 @@ providers with different ``base_dn``. The value of this option must be a valid
...
 
-When this option is used, ``dn_string`` has to be updated accordingly. Following
-the previous example, if your users have the following two DN:
-``dc=companyA,dc=example,dc=com`` and ``dc=companyB,dc=example,dc=com``, then
-``dn_string`` should be ``dc=example,dc=com``. If the ``query_string`` option is
-``uid="{username}"``, then the authentication provider can authenticate users
-from both DN.
+When this option is used, ``query_string`` will search in the DN specified by ``dn_string``
+and the DN resulted of the ``query_string`` will be used to authenticate the user with
+his password.
```
Comment (lsmith77) [inline security/ldap.rst]: his -> their

Comment (lsmith77) [reply security/ldap.rst]: I think keeping an example makes sense

Comment (OskarStark) [reply security/ldap.rst]: Yes but let’s use the new sentence as introduction before the example

Comment (Simperfit) [reply security/ldap.rst]: changed

**Comment by Simperfit** [comment]
another review ?

**Comment by javiereguiluz** [comment]
Thanks for improving the wording of this option. Merged!

**Pull request closed without merging**